libraries/BluetoothSerial/src/BTAddress.cpp

```diff
@@ -86,19 +86,25 @@ esp_bd_addr_t *BTAddress::getNative() const {
 
 /**
  * @brief Convert a BT address to a string.
- *
- * A string representation of an address is in the format:
- *
+ * @param [in] capital changes the letter size
+ * By default the parameter `capital` == false and the string representation of an address is in the format:
  * ```
  * xx:xx:xx:xx:xx:xx
  * ```
- *
+ * When the parameter `capital` == true the format uses capital letters:
+ * ```
+ * XX:XX:XX:XX:XX:XX
+ * ```
  * @return The string representation of the address.
  */
-std::string BTAddress::toString() const {
+std::string BTAddress::toString(bool capital) const {
```

Comment:
I see that originally the method returns `std::string`, but please change it to Arduino `String` instead.